docs/src/rules/array-callback-return.md

```diff
@@ -35,6 +35,7 @@ This rule finds callback functions of the following methods, then checks usage o
 * [`Array.prototype.reduceRight`](https://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.reduceright)
 * [`Array.prototype.some`](https://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.some)
 * [`Array.prototype.sort`](https://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.sort)
+* [`Array.prototype.toSorted`](https://tc39.es/proposal-change-array-by-copy/#sec-array.prototype.toSorted)
```

Comment:
https://github.com/tc39/ecma262/pull/2997 hasn't been merged yet, so we cannot link to the ECMA-262.

Reply:
Maybe we should wait for https://github.com/tc39/ecma262/pull/2997 to be merged into the spec so that the new feature becomes official before merging this PR?

Reply:
The feature is official the moment the committee grants it consensus; merging into the spec is ceremonial.

Reply:
I don't see any reason to wait for the formal spec merge. We can always update the docs later.